### DISCUSSION TITLE
vpn和proxy模式的区别

### DISCUSSION THREAD

**Discussion by robin12jbj** [2dust/v2rayNG#1483] [Q&A]
还是不太清楚v2rayNG的vpn和代理模式的区别，vpn模式是有内置DNS吗？

**Comment by Ray1735** [comment]
同问

**Reply by robin12jbj** [reply]
个人猜测：vpn模式可以进行全局代理，可以接收例如Terminal的命令，但proxy只能代理http之类的

**Comment by 2dust** [comment]
https://github.com/2dust/v2rayNG/wiki/Mode

**Reply by upsangel** [reply]
@2dust 請問 https://github.com/2dust/v2rayNG/wiki/Mode 中提到的 HTTP proxy 127.0.0.1:10809 是否在最新 android 版中依然是可行？

遇到的情況是 設置爲 proxy mode 後，節點可以正常返回延遲，但是 在android系統WIFI 設定中 無論輸入 127.0.0.1 的 10808 端口還是 10809 端口，在chrome和youtube瀏覽器中沒有任何代理效果，都是直出。但是調整爲 VPN mode 就可以代理。

爲什麼不選擇 vpn mode？因爲同時想使用 tailscale vpn，利用tailscale 的 100.xxx.xxx.xxx 的 IP 連到 vless 節點。但是 tailscale 開啓 app後，不能開啓 v2rayng 的 vpn模式，只能二選一。

**Reply by upsangel** [reply]
補充： 嘗試 firefox nightly 使用 10808 端口，是可以正常轉發proxy。 現時問題是， http proxy 是否在 10809 監聽？ （android 版）

**Reply by 2dust** [reply]
最新版本 HTTP 和 SOCKS 已经合并了，都是设置中的值，默认为10808 。
wiki 现在已经改正。
